src/Toolkit/Dom.php

```diff
@@ -295,6 +295,28 @@ public static function isAllowedUrl(string $url, array $options)
             return 'Invalid data URI';
         }
 
+        // allow valid email addresses
+        if (Str::startsWith($url, 'mailto:') === true) {
+            $address = Str::after($url, 'mailto:');
+
+            if (empty($address) || V::email($address) === true) {
+                return true;
+            }
+
+            return 'Invalid email address';
+        }
+
+        // allow valid telephone numbers
+        if (Str::startsWith($url, 'tel:') === true) {
+            $address = Str::after($url, 'tel:');
+
+            if (empty($address) || preg_match('!^(\+|)[0-9]+$!', $address)) {
```

Comment:
I believe this can be written like this for better readability.

```suggestion
            if (empty($address) || preg_match('!^[+]?[0-9]+$!', $address)) {
```

In theory `\+?` should also be possible, but it's also hard to read.